napari/_vispy/vispy_surface_layer.py

```diff
@@ -53,11 +53,15 @@ def _on_data_change(self, event=None):
         self._on_translate_change()
 
     def _on_colormap_change(self, event=None):
-        cmap = self.layer.colormap[1]
         if self.layer.gamma != 1:
```

Comment:
I think this check is no longer necessary after #1456, now that gamma is performed by the shader?  Can you check to see if it works to just assign `cmap = Colormap(*self.layer.colormap)` without the gamma check?